Application/EdFi.Ods.AdminApi/dbadmin.Dockerfile

```diff
@@ -10,16 +10,13 @@ ENV POSTGRES_USER=${POSTGRES_USER}
 ENV POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
 ENV POSTGRES_DB=postgres
 
-ENV ADMINAPP_DATABASE_VERSION="2.4.26"
-
-COPY run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
+COPY Docker/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
+COPY Artifacts/PgSql/Structure/Admin/ /tmp/AdminApiScripts/
```

Comment:
Copying SQL scripts directly into the container instead of loading from NuGet. To do so, I needed to move the file up into a parent directory. Hence line 15 (now 13) also changed.